docs/reference/ilm/apis/remove-policy-from-index.asciidoc

```diff
@@ -1,17 +1,18 @@
 [role="xpack"]
 [testenv="basic"]
 [[ilm-remove-policy]]
-=== Remove policy from index API
+=== Remove lifecycle policy API
```

Comment:
I think this is ambiguous, by calling it the "Remove lifecycle policy API" it makes it sound like this removes the actual policy, maybe something like "Remove lifecycle policy from index API" or "Detach lifecycle policy API"?

Reply:
Thanks for the feedback @dakrone. I was trying to avoid confusion that the API only target indices, but I agree that this change may cause a more important confusion. I reverted the change.